src/libsync/configfile.cpp

```diff
@@ -245,6 +246,12 @@ qint64 ConfigFile::maxChunkSize() const
     return settings.value(QLatin1String(maxChunkSizeC), 1000 * 1000 * 1000).toLongLong(); // default to 1000 MB
 }
 
+qint64 ConfigFile::failsafeMaxChunkSize() const
+{
+    QSettings settings(configFile(), QSettings::IniFormat);
+    return settings.value(QLatin1String(failsafeMaxChunkSizeC), 100 * 1000 * 1000).toLongLong(); // default to 100 MB
```

Comment:
It's best to have this `100 * 1000 * 1000` as a const variable somewhere, as we are repeating it. This is asking for an error at some point.

Reply:
Fully agree. Have turned defaults of all chunk sizes to global const vars... especially tests depend on it to work correctly.